src/main/kotlin/com/jetbrains/snakecharm/codeInsight/SnakemakeSectionsInShellReferenceContributor.kt

```diff
@@ -24,20 +24,28 @@ class SnakemakeParamsInShellReferenceContributor : PsiReferenceContributor() {
             .inside(PyCallExpression::class.java)
             .inside(SmkRunSection::class.java)
 
+    private val allowedInShellWithKeywords = setOf(
+            SnakemakeNames.SECTION_PARAMS,
+            SnakemakeNames.SECTION_INPUT,
+            SnakemakeNames.SECTION_OUTPUT,
+            SnakemakeNames.SECTION_RESOURCES,
+            SnakemakeNames.SECTION_LOG
+    )
```

Comment:
How did you get this list of `allowed` sections? Is it mentioned in snakemake documentation our somewhere in snakemake source code?